### ISSUE TITLE
Don't leave low level companions behind

### ISSUE THREAD

**Issue by pingje** [TLDRMissions/TLDRMissions#253]
**TLDR Version of your idea**
Bonus XP missions should still be dispatched when there are fewer than 4 low level companions.

**Slightly longer version**
Special handling for low level companions should still work if you have fewer than 4 companions under 60. All but one of my companions are max level. 1 is 40. It is not getting dispatched on missions, and so isn't getting leveled.

**Additional context**
I don't run Torghast much on my alts. I recently did a run for the Anima Salvage calling, and found Ella in the process. Ella is now level 40 from a few random tokens, but they are the _only_ companion I have under 60, and they are not getting sent on missions because (I assume) it doesn't meet the threshold for bonus XP handling.

If I re-prioritize for follower XP, with special handling unchecked, Ella has no chance on fewest followers, so my current workaround is to turn off automatic sending, set it to use the most troops, send the first one with Ella, then flip back to automatic sending with fewest followers for the rest.

**Comment by teelolws** [comment]
I had other users complain that they felt it was a waste to send those missions out with fewer than 4 followers. I can consider adding another option so you can decide the minimum followers to send.

**Comment by pingje** [comment]
An option like that would work for my case. I have a couple of tables that are in this situation, so it would help.

**Comment by teelolws** [comment]
I don't have the followers to test it. Can you try this? Look for the dropdown next to the special treatment button.
[TLDRMissions.zip](https://github.com/TLDRMissions/TLDRMissions/files/8758338/TLDRMissions.zip)

**Comment by pingje** [comment]
I see the option, and I was able to test it. Worked perfectly, thank you!
